frontend/test/pages/SettingsPage.ts

```diff
@@ -0,0 +1,49 @@
+import { By, Locator, WebElement, until, Select } from 'selenium-webdriver'
+import { BasePage } from './BasePage'
+
+export class SettingsPage extends BasePage {
+  override baseurl = 'http://localhost:8080/simulator/settings'
+  override visibleLocator: Locator = By.xpath("//h3[contains(text(), 'Settings')]")
+
+  async openNewValidatorModal() {
+    const locator = By.xpath(`//button[contains(text(), 'New Validator')]`)
+    return this.driver.wait(until.elementLocated(locator), 5000).click()
+  }
+  async getValidatorsElements(): Promise<WebElement[]> {
+    return this.driver.findElements(By.xpath("//div[@data-testid = 'validator-item-container']"))
+  }
+
+  async createValidator() {
```

Comment:
@bradleySuira this method should receive the provider, model, and stake so it is more reliable when testing the update validator

Reply:
fixed now 
we have all params in the util method
```
 async createValidator({
    provider,
    model,
    stake
  }: {
    provider: string
    model: string
    stake: number
  })
```